apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/DataSourceWrapper.java

```diff
@@ -120,7 +125,7 @@ public DataSourceWrapper(DbProperties dbProperties) {
         String dbUrl = dbProperties.getDbUrl();
         if (StringUtils.isBlank(dbUrl)) {
             String dbFileName = dbProperties.getDbFileName();
-            dbUrl = String.format("jdbc:%s:%s;%s", dbProperties.getDbType(), dbProperties.getDbDir() + "/" + dbFileName, dbProperties.getDbParams());
+            dbUrl = String.format("jdbc:%s:file:%s;%s", dbProperties.getDbType(), dbProperties.getDbDir() + "/" + dbFileName, dbProperties.getDbParams());
```

Comment:
Let's define it explicitly as **file** because there different values are possible here:
jdbc:h2:mem:...
jdbc:h2:file:...
jdbc:h2:tcp:...
jdbc:h2:ssl:...
jdbc:h2:zip:....
jdbc:h2:<url>....